templates/docs/examples/patterns/tabs/tab-buttons.html

```diff
@@ -0,0 +1,97 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Tabs / Tab Buttons{% endblock %}
+
+{% block standalone_css %}patterns_tab-buttons{% endblock %}
+
+{% block content %}
+
+<div class="p-tab-buttons">
+  <div class="p-tab-buttons__list" role="tablist" aria-label="Juju technology">
+    <button
+      class="p-tab-buttons__button has-icon"
+      role="tab" 
+      aria-selected="true" 
+      aria-controls="olm-tab" 
+      id="olm"
+    >
+      <i class="p-icon--information"></i><span>Charmed Operator Lifecycle Manager</span>
+    </button>
+    <button
+      class="p-tab-buttons__button"
+      role="tab"
+      aria-selected="false"
+      aria-controls="operator-tab"
+      id="operator" 
+      tabindex="-1"
+    >Charmed Operator SDK</button>
+    <button
+      class="p-tab-buttons__button"
+      role="tab"
+      aria-selected="false"
+      aria-controls="charmhub-tab"
+      id="charmhub"
+      tabindex="-1"
+    >Charmhub</button>
+  </div>
+</div>
+
+<div tabindex="0" role="tabpanel" id="olm-tab" aria-labelledby="olm">
+  <p>A system to help you move from configuration management to application management across your hybrid cloud estate - through sharable, reusable, tiny applications called Charmed Operators.</p>
+</div>
+
+<div tabindex="0" role="tabpanel" id="operator-tab" aria-labelledby="operator" hidden="hidden">
+  <p>A set of tools to help you write Charmed Operators and to package them as Charms.</p>
+</div>
+
+<div tabindex="0" role="tabpanel" id="charmhub-tab" aria-labelledby="complex" hidden="hidden">
+  <p>A repository for charms - from Observability to Data to Identity and more.</p>
+</div>
+
+<div class="p-tab-buttons is-dense">
```

Comment:
I guess we should maybe keep this in separate example as different variant ("kind", "size" or something 😉).